### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by JGon26**
Added GitHub sign-in method, won't sign you into GitHub if your email is already used with another sign-in method.
Added styling to buttons
converted buttons to jsx components

### CODE REVIEW

File: components/FacebookSignIn.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          >
+            <div className="facebook-btn">
+              <img
+                className="facebook-logo"
+                src="/facebook_logo.png"
+                alt="facebook"
+                width="25px"
+                height="25px"
+              />
+              <h3 className="facebook-signin">
```
Comment (Darumin) [inline components/FacebookSignIn.js:19]: This might affect accessibility later? Not sure though.

File: pages/login/index.js [hunk 2]
```diff
@@ -1,34 +1,23 @@
...
-            />
-            <h3 className="google-signin">
-              Continue with Google
-            </h3>
-          </div>
-        </button>
-        <button type="button" className="facebook-btn" onClick={signInWithFacebook}>Continue with Facebook</button>
+      <div className="login-btn-container">
+        <div id="login">
+          <p>Sign up today to save your better <br /> and best beans</p>
```
Comment (Darumin) [inline pages/login/index.js:16]: So clean!

**Review by Darumin — COMMENTED** [review]
Everything looks good to me.

**Review by zhuzhuxia078 — APPROVED** [review]
looks good

**Pull request merged**